charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-code-kubeflow-ConfigMap.yaml

```diff
@@ -245,11 +245,10 @@ data:
     \n\nif __name__ == \"__main__\":\n    main()"
 kind: ConfigMap
 metadata:
-  annotations: {}
   labels:
     app: kubeflow-pipelines-profile-controller
     app.kubernetes.io/component: ml-pipeline
     app.kubernetes.io/name: kubeflow-pipelines
     application-crd-id: kubeflow-pipelines
-  name: kubeflow-pipelines-profile-controller-code-dhffkh7f55
+  name: kubeflow-pipelines-profile-controller-code
```

Comment:
how come the suffix is getting removed here, but not in `charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/kubeflow-pipelines-profile-controller-env-5252m69c4c-kubeflow-ConfigMap.yaml`

Reply:
https://github.com/kubernetes-sigs/kustomize/issues/5047 again seems an issue from kustomize version

Reply:
in that case, can you check why is it happening only in rds-s3 and not in vanilla or rds-only

Reply:
you are right, both rds-only and vanilla have the hash-suffix